<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EssayEditor</class>
 <widget class="QWidget" name="EssayEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>610</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="toolbars">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QFrame" name="toolbar1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="KComboBox" name="styleBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentText">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Pick a style...</string>
          </property>
          <property name="autoCompletion">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Normal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Heading 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Heading 2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Heading 3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Heading 4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Heading 5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Heading 6</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="KContextualHelpButton" name="mainToolbarHelp">
          <property name="contextualHelpText">
           <string>This is the main toolbar, where you can customized how text is formatted.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KFontRequester" name="fontBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Text Font</string>
          </property>
          <property name="title">
           <string>Text Font</string>
          </property>
          <property name="sampleText">
           <string>The lazy fox...</string>
          </property>
          <property name="font">
           <font>
            <family>PT Serif</family>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="boldButton">
          <property name="toolTip">
           <string>Bold</string>
          </property>
          <property name="text">
           <string>Bold</string>
          </property>
          <property name="icon">
           <iconset theme="format-text-bold"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::ToolButtonPopupMode::DelayedPopup</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="italicButton">
          <property name="toolTip">
           <string>Italic</string>
          </property>
          <property name="text">
           <string>Italic</string>
          </property>
          <property name="icon">
           <iconset theme="format-text-italic"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="underLineButton">
          <property name="toolTip">
           <string>Underline</string>
          </property>
          <property name="text">
           <string>Underline</string>
          </property>
          <property name="icon">
           <iconset theme="format-text-underline"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="strikeoutButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="format-text-strikethrough"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="subScriptButton">
          <property name="text">
           <string>SubScript</string>
          </property>
          <property name="icon">
           <iconset theme="format-text-subscript"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="superScriptButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="format-text-superscript"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KColorButton" name="textColor">
          <property name="toolTip">
           <string>Text Color</string>
          </property>
          <property name="text">
           <string>Text Color</string>
          </property>
          <property name="icon">
           <iconset theme="color-picker"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KColorButton" name="bgColor">
          <property name="alphaChannelEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignLeft">
      <widget class="QFrame" name="toolbar2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="alignLeftButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="format-justify-left"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="alignCenterButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="format-justify-center"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="alignRightButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="format-justify-right"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="alignJustifyButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="format-justify-fill"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="indentLessButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="format-indent-less"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="indentMoreButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="format-indent-more"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLineEdit" name="documentTitleText">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="placeholderText">
           <string>Document Title</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="spacingLabel">
          <property name="text">
           <string>Spacing: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KHistoryComboBox" name="spacingBox">
          <item>
           <property name="text">
            <string>Single</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Double</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </item>
   <item>
    <widget class="KRichTextEdit" name="editorBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KHistoryComboBox</class>
   <extends>KComboBox</extends>
   <header>khistorycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KRichTextEdit</class>
   <extends>KTextEdit</extends>
   <header>krichtextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KContextualHelpButton</class>
   <extends>QToolButton</extends>
   <header>kcontextualhelpbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KFontRequester</class>
   <extends>QWidget</extends>
   <header>kfontrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
